### PR TITLE
Upgrade Laravel packages

### LINKED ISSUE #57: Laravel Upgrade
Update Laravel to 8.x

### PR DESCRIPTION
**Pull request by dhawton**
Closes #57

### CODE REVIEW

File: app/Providers/RouteServiceProvider.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ protected function mapWebRoutes()
     protected function mapApiRoutes()
     {
         if (env('APP_ENV') == "dev") {
-            Route::domain("api.vatusa.devel")
+            Route::domain("api.vatusa.hawton.dev")
```
Comment (bnahin) [inline app/Providers/RouteServiceProvider.php:69]: Let's make this domain come from ENV

File: app/Providers/RouteServiceProvider.php [hunk 2]
```diff
@@ -91,7 +91,7 @@ protected function mapApiRoutes()
     protected function mapLoginRoutes()
     {
         if (env('APP_ENV') == "dev") {
-            Route::domain("login.vatusa.devel")
+            Route::domain("login.vatusa.hawton.dev")
```
Comment (bnahin) [inline app/Providers/RouteServiceProvider.php:94]: Let's make this domain come from ENV

**Review by bnahin — CHANGES_REQUESTED** [review]

File: app/Providers/RouteServiceProvider.php (hunk 1 above)
Comment (dhawton) [reply app/Providers/RouteServiceProvider.php:69]: That wasn't even supposed to be in the commit.. dang it..

Comment (dhawton) [reply app/Providers/RouteServiceProvider.php]: Resolved in 6a45112

File: app/Providers/RouteServiceProvider.php (hunk 2 above)
Comment (dhawton) [reply app/Providers/RouteServiceProvider.php]: Resolved in 6a45112

**Review by bnahin — APPROVED** [review]

**Pull request merged**